src/algebraic_geometry/is_open_comap_C.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import algebraic_geometry.prime_spectrum
+import ring_theory.polynomial.basic
+/-!
+The morphism `Spec R[x] --> Spec R` induced by the natural inclusion `R --> R[x]` is an open map.
+-/
+
```

Comment:
currently, all these names are added to the root namespace, and some of them sound terribly generic :stuck_out_tongue_winking_eye: 

Would it make sense to start the file with
```
namespace algebraic_geometry
namespace polynomial
```

Alternatively, maybe even more of these lemmas should be stated without the language of `prime_spectrum`.
I think a lot of them are actual "pure" commutative algebra.